onnxruntime/core/providers/cpu/cpu_execution_provider.cc

```diff
@@ -790,8 +790,6 @@ class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 17, ST
 // !!PLEASE READ BELOW!! Following that, add new entries above this comment
 
 /*  *** IMPORTANT! ***
- If kernel registrations are incorrectly updated, ORT format models get broken as the kernel hashes may be invalidated.
```

Comment:
@skottmckay should this be replaced another comment? I think it's still important to correctly register kernels

Reply:
How would they register it correctly if the kernel def builder no longer generates a hash?